### ISSUE TITLE
overall strategy for consistency (predictable behaviour)

### ISSUE THREAD

**Issue by bdolor** [BCcampus/bcc-sage#212]
## Submit a feature request or bug report

- [ ] This is a feature request
- [ ] This is a bug report
- [x] This request isn't a duplicate of an [existing issue](https://github.com/BCcampus/bcc-sage/issues)
- [ ] You cleared your Web Browsers cache to ensure all new styling and images were loaded. 

---

**What is the current behavior?**

- Sometimes images are linked, sometimes they are not.
- Sometimes link colours are purple, white, black or blue. Sometimes text without links is purple, black, or white. 
- Sometimes hover states for links are underlined, sometimes they are not. 
- Sometimes hover states for links are black, white, or blue. 

**What is the expected or desired behavior?**

We need a consistent approach to avoid unpredictable behaviour. 

1. linked images vs non-linked images
2. link colour
3. link hover states

reference: https://webaim.org/techniques/hypertext/link_text#appearance

> Links should look like links, and nothing else should. Users may get frustrated if they try to click on textual phrases or graphics that look like links but are not. They will also be frustrated if they have to move their mouse all over the page trying to discover links that do not look like links.

**Comment by murphybarb** [comment]
Proposed overall strategy for consistency: 
- yes to all linked images
- link colour – white for text on a high contrast coloured background (i.e. purple). All other text links – purple and blue 
- link hover states 
underline
**White text on background** – blue on hover
**blue and purple text** – black on hover


Header: 
Top main navigation – can we leave that black and use an underline as indicated in the orginal design mockup. 

Footer: 
"Quick Links" - become purple text 

![screen_shot_2018-10-24_at_11_42_54_am](https://user-images.githubusercontent.com/11672469/47453830-83196500-d782-11e8-966e-9c26bb44fcdd.png)

**Comment by bdolor** [comment]
menu items will be purple with a blue hover state and underlined.

**Comment by bdolor** [comment]
I think most of this is done, but would benefit from a second set of eyes. Please validate.

**Comment by ehbeattie** [comment]
This looks good, but wonder if you are able to add some coding to create an underline of the text when a user hovers over said text in an image?

**Comment by bdolor** [comment]
complete via https://github.com/BCcampus/bcc-sage/commit/f31180c0d63a0e9d1755c119b0c6707131200c47
